resources/views/customer/receipt/print.blade.php

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Receipt</title>
+
+    <style>
+        .page-break {
+            page-break-after: always;
+        }
+
+        .text-center {
+            text-align: center;
+        }
+
+        .text-right {
+            text-align: right;
+        }
+
+        table {
+            border-collapse: collapse;
+            width: 100%;
+        }
+
+        table, th, td {
+            border: 1px solid lightgray;
+        }
+
+    </style>
+</head>
+<body>
+    <h1>Customers</h1>
```

Comment:
Can you add who the customer is. The same for other receipt types and bills (vendors). You may put it at the top.
Not necessary to make the same change on email for now.